Sources/OpenTelemetryApi/Context/ActivityContextManager.swift

```diff
@@ -4,6 +4,7 @@
  */
 
```

Comment:
rather than changing the existing context manager, a separate one should be added for linux.

Reply:
I believe the bug this was addressing has been fixed upstream so this change can probably be discarded now